README.md

```diff
@@ -3,12 +3,35 @@
 # Xlang
 
 ## Overview
-Xlang is a system that enables a component written in any supported language to be callable from any other supported language using object-oriented programming constructs.  
+The xlang project enables developers to take existing shared libraries,
+implemented in one programming language,
+and make that library's APIs available to client code using a different programming language.
+Thus the name "xlang," for cross-language.
+
+Additionally, all of the xlang tooling will be platform-agnostic therefore will work on your favorite operating system.
+This means that if your shared library is platform-agnostic (i.e. portable to various operating systems),
+then you can use the xlang tooling to make that shared library available to various client programming language on those various platforms.
+
+A *hypothetical* example might be taking the Python numpy library and making it available to C# applications running on Linux.
+
+We're a *long* way was achieving that ultimate goal. We are in the very, very early stages of starting the project.
+However, we want and encourage community feedback and contributions. As such, we decided to do *all* of the project development in the open on GitHub.
+We look forward to your thoughts on the project.
+
+## What xlang is NOT
+
+* The xlang project is not a port of the Windows Runtime, COM, DCOM or related technology.
+  * That said, many of the Microsoft members of the xlang team designed the Windows Runtime. As a result, we hope to incorporate many of the learnings from that prior project.
```

Comment:
consider removing. At best, most comment that this project borrows many concepts from these technologies.